app/services/decisions/capital_decision_tree.rb

```diff
@@ -2,14 +2,17 @@ module Decisions
   class CapitalDecisionTree < BaseDecisionTree
     def destination
       case step_name
+      when :property_type
+        # TODO: Add next step
+        edit(:saving_type)
       when :saving_type
         after_saving_type(form_object.saving)
       when :savings
         # TODO: Add next step
-      when :property_type
-        # TODO: Add next step
+        edit(:premium_bonds)
```

Comment:
Be good to add an explicit TODO comment where a destination has been added in order to get the flow working.